app/commands/decidim/extra_user_fields/admin/export_users.rb

```diff
@@ -7,9 +7,10 @@ module Admin
       class ExportUsers < Rectify::Command
         # format - a string representing the export format
         # current_user - the user performing the action
-        def initialize(format, current_user)
+        def initialize(format, current_user, current_organization)
```

Comment:
No need to add `current_organization` since it is already accessible directly with `current_organization` (see method `collection` at the end of file) or accessible with `current_user.organization`